sdk/search/Azure.Search.Documents/src/Spatial/SpatialFormatter.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Text;
+using Azure.Core;
+#if EXPERIMENTAL_SPATIAL
+using Azure.Core.GeoJson;
+#endif
+
+namespace Azure.Search.Documents
+{
+    /// <summary>
+    /// Encodes geometry from both Azure.Core and Microsoft.Spatial for use in OData filters.
+    /// </summary>
+    internal static class SpatialFormatter
+    {
+        /// <summary>
+        /// Encodes the longitude and latitude of points or positions for use in OData filters.
+        /// </summary>
+        /// <param name="longitude">The longitude to encode, which may also be known as Y.</param>
+        /// <param name="latitude">The latitude to encode, which may also be known as X.</param>
+        /// <returns>The OData filter-encoded POINT string.</returns>
+        public static string EncodePoint(double longitude, double latitude)
```

Comment:
Did you consider using https://docs.microsoft.com/en-us/dotnet/api/microsoft.spatial.wellknowntextsqlformatter?view=odata-spatial-7.0 ?

Reply:
Thanks. I didn't actually know about that, but given we have to handle the code for GeoJson anyway, I'd rather keep it isolated in the same spot rather than use reflection to call this. Besides, once we build GeoJson in with Azure.Core, per my TODO comment we can reduce the duplication here anyway if we use adapters instead of proxies.